recipes/cyrus-sasl/all/conanfile.py

```diff
@@ -0,0 +1,199 @@
+import glob
+import os
+
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class CyrusSaslConan(ConanFile):
+    name = "cyrus-sasl"
+    license = "BSD-4-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.cyrusimap.org/sasl/"
+    description = (
+        "This is the Cyrus SASL API implementation. "
+        "It can be used on the client or server side "
+        "to provide authentication and authorization services."
+    )
+
+    topics = ("SASL", "authentication", "authorization")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_openssl": [True, False],
+        "with_cram": [True, False],
+        "with_digest": [True, False],
+        "with_scram": [True, False],
+        "with_otp": [True, False],
+        "with_krb4": [True, False],
+        "with_gssapi": [True, False],
+        "with_plain": [True, False],
+        "with_anon": [True, False],
+        "with_postgresql": [True, False],
+        "with_mysql": [True, False],
+        "with_sqlite3": [True, False],
+    }
+    default_options = {
+        "shared": True,
+        "fPIC": True,
+        "with_openssl": True,
+        "with_cram": True,
+        "with_digest": True,
+        "with_scram": True,
+        "with_otp": True,
+        "with_krb4": True,
+        "with_gssapi": True,
+        "with_plain": True,
+        "with_anon": True,
+        "with_postgresql": False,
+        "with_mysql": False,
+        "with_sqlite3": False,
+    }
+    exports_sources = ("patches/**",)
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
```

Comment:
```suggestion
```

Removing settings is part of configure method, not config options. please, read the documentation: https://docs.conan.io/en/latest/reference/conanfile/methods.html#configure-config-options

Reply:
Thank you for your guidance :+1: